apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1165,6 +1166,10 @@ function validateAssessment({
   // instances that instructors will never touch again, as they won't benefit
   // from fixing things. So, we'll only show some warnings for course instances
   // which are accessible either now or any time in the future.
+
+  // NOTE (@reteps): We are deprecating `allowAccess` in favor of `publishing`.
+  // So further validation will never be done here.
+
```

Comment:
Formatting nits to make it clear what these comments are associated with:

```suggestion
  //
  // NOTE (@reteps): We are deprecating `allowAccess` in favor of `publishing`.
  // So further validation will never be done here.
```